lib/origen/sub_blocks.rb

```diff
@@ -389,6 +401,10 @@ def get_sub_block(name)
       sub_blocks[name]
     end
 
+    def get_sub_block_group(name)
+      sub_block_groups[name]
+    end
```

Comment:
Why is this method required?

Reply:
This is what the singleton method for sub block groups calls. Just doing similar to what is done for sub blocks. I guess I could have put it in the singleton method directly -- just was using the same structure as for sub blocks.

Reply:
Ok.
Generally I don't like methods called `get_` as part of a user API, not very Rubyish, and in this case even less when its a straight wrapper around an existing accessor, `sub_block_groups`.
Also looking above, I can see this method being called on line 377, and then the unwrapped version being used on line 389 (which is fine), so overall I think we should not be adding this redundant method here.

Reply:
Was just emulating what was there already-- I can clean them both up if you like.